src/Compilers/Core/Portable/CodeGen/ITokenDeferral.cs

```diff
@@ -21,6 +21,8 @@ internal interface ITokenDeferral
 
         Cci.IFieldReference GetArrayCachingFieldForConstants(ImmutableArray<ConstantValue> constants, Cci.IArrayTypeReference arrayType, SyntaxNode syntaxNode, DiagnosticBag diagnostics);
 
+        Cci.IFieldReference? GetFieldForDataString(string text, ImmutableArray<byte> data, SyntaxNode syntaxNode, DiagnosticBag diagnostics);
```

Comment:
>GetFieldForDataString [](http://example.com/codeflow?start=29&length=21)

It looks like this API is not consumed from any shared code. Therefore, it looks unnecessary to have it in this interface.